### PR TITLE
Delay fee changes

### PR DESCRIPTION
**Pull request by rustyrussell**

### CODE REVIEW

File: doc/lightning-setchannelfee.7.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ lightning-setchannelfee -- Command for setting specific routing fees on a lightn
 SYNOPSIS
 --------
 
-**setchannelfee** *id* \[*base*\] \[*ppm*\]
+**setchannelfee** *id* \[*base*\] \[*ppm*\] [\*enforcedelay\*]
```
Comment (vincenzopalazzo) [inline doc/lightning-setchannelfee.7.md:7]:
```suggestion
**setchannelfee** *id* \[*base*\] \[*ppm*\] \[*enforcedelay*\]
```

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM https://github.com/ElementsProject/lightning/pull/4806/commits/7871e8b1be377cdff28f3fbe6912f452b1b8f932

Only a small change request on the doc, the sequence between `\[` and `\*` it is always tricky

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4806/commits/25acf5a46f953e6610c594f4e8dffa977fd7df94

**Review by cdecker — APPROVED** [review]
ACK 25acf5a46f953e6610c594f4e8dffa977fd7df94

**Pull request merged**